### ISSUE TITLE
Can we have possibility to exclude some projects from workspace

### ISSUE THREAD

**Issue by EnisBiberovic** [JaneySprings/DotRush#6]
It would be good if there were a configuration option to exclude some projects by matching the csproj filename with a pattern.

**Comment by JaneySprings** [comment]
Hi @EnisBiberovic ! It has a `dotrush.roslyn.projectFiles` option:
https://github.com/JaneySprings/DotRush/blob/07c966c4b2ab2dbe2f22e3a178a80f78847e5c01/.vscode/settings.json#L4-L7

Is this enough for your scenario?

**Comment by EnisBiberovic** [comment]
It is good when the number of projects is small, I have a situation with a large number of projects where the main build system is maven. That built system creates some temporary projects that I want to exclude.

Your extension is awesome, and working well, but it would be good if it is possible to exclude projects by pattern, so that "projects generated and used by maven plugins" would not be processed.

Since a lot of c# projects file exists, the projects are grouped into solutions, and projects that are designed for VS have Reference instead of ProjectReference, because of very pure VS performances when have a large number of the projects.

I hope you got a better understanding of the use case.

**Comment by JaneySprings** [comment]
OK, I will add this feature as soon as possible!

**Comment by EnisBiberovic** [comment]
Thanks.

**Comment by JaneySprings** [comment]
Fixed in `24.10.283`. Now you can specify the `dotrush.roslyn.excludePatterns` option, for example:
```json
{
    "dotrush.roslyn.excludePatterns": [
        "**/*generated.csproj"
    ]
}
```

**Comment by EnisBiberovic** [comment]
Thank you very much
